Dockerfile.local

```diff
@@ -0,0 +1,60 @@
+# Build the manager binary
+FROM golang:1.14.1 as builder
+
+ARG service_alias
+# The tuple of controller image version information
+ARG service_controller_git_version
+ARG service_controller_git_commit
+ARG build_date
+# The directory within the builder container into which we will copy our
+# service controller code.
+ARG work_dir=/github.com/aws-controllers-k8s/$service_alias-controller
+WORKDIR $work_dir
+# For building Go Module required
+ENV GOPROXY=https://proxy.golang.org,direct
+ENV GO111MODULE=on
+ENV GOARCH=amd64
+ENV GOOS=linux
+ENV CGO_ENABLED=0
+ENV VERSION_PKG=github.com/aws-controllers-k8s/$service_alias-controller/pkg/version
+# Copy the Go Modules manifests and LICENSE/ATTRIBUTION
+COPY $service_alias-controller/LICENSE $work_dir/LICENSE
+COPY $service_alias-controller/ATTRIBUTION.md $work_dir/ATTRIBUTION.md
+# use local mod files
+COPY $service_alias-controller/go.local.mod $work_dir/go.local.mod
```

Comment:
Go compiler doesn't recognize `go.local.mod` files, it should be copied as `go.mod`:
`COPY $service_alias-controller/go.local.mod $work_dir/go.mod`

Reply:
The line below:

`-modfile="${work_dir}/go.local.mod"`

makes this work.

Reply:
Ahhh, thanks for the pointer TIL!